game/scripts/npc/abilities/bane_brain_sap.txt

```diff
@@ -37,8 +37,8 @@
     {
       "brain_sap_damage" //OAA
       {
-        "value"                                           "75 150 225 300 600 900"
-        "special_bonus_unique_bane_2"                     "+300"
+        "value"                                           "100 200 300 400 800 1200"
```

Comment:
why?

Reply:
+300 Brain Sap damage looked like a crap talent, I wanted to make it +400. 400 = 400. Bane bad damage good